Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM alpine:3.7
-RUN apk add --no-cache ca-certificates
-RUN adduser -D -g '' vault-manager
-USER vault-manager
-COPY vault-manager query.graphql /
+FROM registry.access.redhat.com/ubi8/go-toolset:latest as builder
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go test ./...
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo ./cmd/vault-manager
```

Comment:
why not making this a `Makefile` target?

Reply:
Good suggestion. Feel free to send a PR :)